src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -844,9 +865,7 @@ describe('SamDebugConfigurationProvider', async function () {
                 },
                 localRoot: appDir,
                 name: input.name,
-                templatePath: pathutil.normalize(
-                    path.join(path.dirname(templatePath.fsPath), 'app___vsctk___template.yaml')
-                ),
+                templatePath: pathutil.normalize(path.join(path.dirname(templatePath.fsPath), 'template.yaml')),
```

Comment:
is this correct? I don't see `buildDir` set in the test input, so shouldn't it create the `app___vsctk___template.yaml` thing?

Reply:
`app___vsctk___template.yaml` now only refers to the generated template in the 'code' target case. So if you don't set `buildDir` it will use the user's template but output to a temporary directory.